src/ui/public/vislib/lib/axis_config.js

```diff
@@ -117,7 +118,7 @@ export default function AxisConfigFactory() {
     };
 
     isPercentage() {
-      return (this._values.vis._attr.mode === 'percentage');
+      return (this._chartConfig.mode === 'percentage');
```

Comment:
I don't see why we would support a single series (a single line for instance) in percentage mode while the rest are not. I feel like percentage mode is more about where the x/y value from each point maps to the axis... but maybe we are both describing different things as series?